tests/package_file_service_spec.rb

```diff
@@ -0,0 +1,131 @@
+describe "Task 1:", host: :localhost do
+  it 'Create the pasture module directories' do
+    file('/etc/puppetlabs/code/environments/production/modules/pasture/manifests')
+      .should be_directory
+    file('/etc/puppetlabs/code/environments/production/modules/pasture/files')
+      .should be_directory
+  end
+end
+
+describe "Task 2:", host: :localhost do
+  it 'Create the pasture main manifest' do
+    file('/etc/puppetlabs/code/environments/production/modules/pasture/manifests/init.pp')
+      .should be_file
+    command('puppet parser validate /etc/puppetlabs/code/environments/production/modules/pasture/manifests/init.pp')
+      .exit_status
+      .should be_zero
+  end
+end
+
+describe "Task 3:", host: :localhost do
+  it 'Update site.pp with classification for pasture.puppet.vm' do
+    file('/etc/puppetlabs/code/environments/production/manifests/site.pp')
+      .content
+      .should match /node\s+(['"])?pasture\.puppet\.vm\1\s+\{.*?include\s+pasture.*?\}/mi
+  end
+end
+
+describe "Task 4:", host: :pasture do
+  it 'Run the agent' do
+    package('pasture')
+      .should be_installed
+      .by('gem')
+  end
+end
+
+describe "Task 5:", host: :pasture do
+  it 'Start the pasture service' do
```

Comment:
@kjhenner Note that I don't use the `process` matcher here because the service must be stopped in a later task. I don't see a way to latch the test into a "passed" state, even if the service is stopped in task 7. As soon as task 7 passes, it would cause this test to fail again (and vice-versa) if I use `process` matchers for each.